test/Main.purs

```diff
@@ -84,13 +84,13 @@ main = do
 runSnapshotTests :: TestArgs -> Aff Unit
 runSnapshotTests { accept, filter } = do
   liftEffect $ Process.chdir $ Path.concat [ "test-snapshots" ]
-  spawnFromParent "spago" [ "build", "-u", "-g corefn" ]
+  spawnFromParent "spago" [ "build", "--purs-args", "-g corefn" ]
   snapshotDir <- liftEffect Process.cwd
-  snapshotPaths <- expandGlobs (Path.concat [ snapshotDir, "snapshots-input" ])
+  snapshotPaths <- expandGlobs (Path.concat [ snapshotDir, "src", "snapshots-input" ])
     [ "Snapshot.*.purs" ]
   schemeBin <- getSchemeBinary
   outputRef <- liftEffect $ Ref.new Map.empty
-  let snapshotsOut = Path.concat [ snapshotDir, "snapshots-output" ]
+  let snapshotsOut = Path.concat [ snapshotDir, "src", "snapshots-output" ]
```

Comment:
Just curious but does `src` need to be here? I think the extra directory is a bit unnecessary.

Reply:
The purescript files need to be in `src` for Spago to consider it a project (this is a new constraint in the new Spago), but the output files do not need to be there as well. I just moved them together with the PureScript sources.

Reply:
The registry has this constraint and Spago just adopts it. The constraint was adopted in the Registry for backwards compatibility with the current situation (where all our tooling expects things to be in `src`) and for consistency, so one does not have to look up a config to find the sources for a given package.